Portal_Perfomance_Employes/Models/Employe.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Portal_Perfomance_Employes.Models
+{
+    public class Employe
```

Comment:
Hi Ygor, could you please correct the spelling of `Employee`?

Reply:
I changed the names to the correct spelling.